custom/icds_reports/views.py

```diff
@@ -1649,3 +1651,15 @@ def post(self, request, *args, **kwargs):
                     messages.success(request, _('Successfully enqueued request to submit files for translations'))
                     return redirect(self.urlname, domain=self.domain)
         return self.get(request, *args, **kwargs)
+
+
+@method_decorator([login_and_domain_required], name='dispatch')
+class InactiveAWW(View):
+    def get(self, request, *args, **kwargs):
+        sync_date = request.GET.get('date', None)
+        if sync_date:
+            sync = IcdsFile.objects.filter(file_added=sync_date).first()
+        else:
+            sync = IcdsFile.objects.filter(data_type='inactive_awws').order_by('-file_added').first()
+        zip_name = 'inactive_awws_%s' % sync.file_added.strftime('%Y-%m-%d')
+        return export_response(sync.get_file_from_blobdb(), 'csv', zip_name)
```

Comment:
It seems possible that the blob will not exist. Do you need to handle `corehq.blobs.exceptions.NotFound` here?